uaclient/api/u/pro/security/vulnerabilities/_common/v1.py

```diff
@@ -328,6 +328,47 @@ def _get_vulnerability_fix_status(
 )
 
 
+class VulnerabilitiesAlreadyFixed:
+    def __init__(self):
+        self.vulns = {}
```

Comment:
```suggestion
        self.vulns = defaultdict(lambda: {"issues": set(), "priority_count": defaultdict(int)})
```

Reply:
(lonely comment left here based on conversation context in other channel)
This is a suggestion to simplify the add function